src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1225,7 +1225,7 @@ void TorrentHandle::forceRecheck()
 
     if (isPaused()) {
         m_nativeHandle.stop_when_ready(true);
-        resume_impl(true, true);
+        resume_impl(false);
```

Comment:
I wonder why the 1st parameter was `true` in the 1st place...

Reply:
@arvidn just to be sure: Do rechecking torrents count against `active_limit`?

Reply:
>I wonder why the 1st parameter was true in the 1st place...

To prevent it from exiting the upload_mode (2nd parameter).